container-storage-setup.sh

```diff
@@ -770,7 +770,7 @@ create_disk_partitions() {
     # signatures on disk or signatures should be wiped. Don't care
     # about any signatures found in the middle of disk after creating
     # partition and wipe signatures if any are found.
-    if ! wipefs -a ${dev}1; then
+    if ! wipefs -f -a ${dev}1; then
```

Comment:
We need to improve changelog. How about following.

`After partition creation, we wipe any signatures on the partition using wipefs. For some reason, for loop devices it was detecting nested partition table signature and wipefs will skip it by default and instead ask user whether to clean it or not. 

Use option -f instead to not ask user the question and delete it alaways.
`